app/pulp/app/tasks/importer.py

```diff
@@ -15,3 +18,25 @@ def delete(repo_name, importer_name):
     :type  importer_name:   str
     """
     models.Importer.objects.filter(name=importer_name, repository__name=repo_name).delete()
+
+
+@shared_task(base=UserFacingTask)
+def sync(repo_name, importer_name):
+    """
+    Call sync on the specified importer.
+
+    Additional "business logic" checks are included here. These checks are distinct from
```

Comment:
What about rewriting this to be specific to this task? Something like:

`This task ensures that the 'feed_url' attribute is set before calling into the sync() method provided by the plugin.`

My thinking is that the statement written now is so general it could be applied to any function that contains business logic.